juju_verify/verifiers/neutron_gateway.py

```diff
@@ -21,42 +21,49 @@
 from juju.unit import Unit
 from packaging.version import Version
 
-from juju_verify.verifiers.base import BaseVerifier, Result, Severity
 from juju_verify.utils.action import data_from_action
 from juju_verify.utils.unit import parse_charm_name, run_action_on_unit
+from juju_verify.verifiers.base import BaseVerifier, Result, Severity
 from juju_verify.verifiers.result import checks_executor
 
 
 class NeutronGateway(BaseVerifier):
     """Implementation of verification checks for the neutron-gateway charm."""
 
-    NAME = 'neutron-gateway'
-    action_name_result_map = {"get-status-routers": "router-list",
-                              "get-status-dhcp": "dhcp-networks",
-                              "get-status-lb": "load-balancers"}
-    action_name_failure_string_map = {"get-status-routers": ("The following routers are "
-                                                             "non-redundant: {}"),
-                                      "get-status-dhcp": ("The following DHCP networks "
-                                                          "are non-redundant: {}"),
-                                      "get-status-lb": ("The following LBaasV2 LBs were "
-                                                        "found: {}. LBaasV2 does not "
-                                                        "offer HA.")}
+    NAME = "neutron-gateway"
+    action_name_result_map = {
+        "get-status-routers": "router-list",
+        "get-status-dhcp": "dhcp-networks",
+        "get-status-lb": "load-balancers",
+    }
+    action_name_failure_string_map = {
+        "get-status-routers": ("The following routers are non-redundant: {}"),
```

Comment:
since they now fit single line, these strings no longer need to be in parenthesis.